apps/web/test/lib/next-config.test.ts

```diff
@@ -0,0 +1,153 @@
+
+import { it, expect, describe, beforeAll, afterAll } from "vitest";
+let userTypeRouteRegExp: RegExp;
+let teamTypeRouteRegExp:RegExp;
+let privateLinkRouteRegExp:RegExp
+let embedUserTypeRouteRegExp:RegExp
+let embedTeamTypeRouteRegExp:RegExp
+
+
+const getRegExpFromNextJsRewriteRegExp = (nextJsRegExp:string) => {
+  // const parts = nextJsRegExp.split(':');
+  
+  // const validNamedGroupRegExp =  parts.map((part, index)=>{
+  //   if (index === 0) {
+  //     return part;
+  //   }
+  //   if (part.match(/^[a-zA-Z0-9]+$/)) {
+  //     return `(?<${part}>[^/]+)`
+  //   }
+  //   part = part.replace(new RegExp('([^(]+)(.*)'), '(?<$1>$2)');
+  //   return part
+  // }).join('');
```

Comment:
Code for automatic conversion. It is tricky in itself. So, no magic in tests for now.